server/views/taskList/caSubmitRefusalTask.pug

```diff
@@ -0,0 +1,22 @@
+- var canSendToDm = licenceStatus.stage === 'ELIGIBILITY' || allowedTransitions.caToDm
+
+div.largeMarginBottom.borderTop
+    h2.heading-medium Submit to decision maker
+
+    div.borderBottomLight.largeMarginBottom
+
+    div.pure-g
+        div.pure-u-3-4
+            if canSendToDm
+                | Ready to submit
+            else if licenceStatus.decisions.postponed
+                | Submission unavailable - HDC application postponed
+            else if licenceStatus.decisions.finalChecksRefused
+                | Submission unavailable - HDC refused
```

Comment:
final checks can't be relevant to this because we can only be in the ELIGIBILITY stage